### PR TITLE
feat(pattern): use cases pattern

### PR DESCRIPTION
**Pull request by sannminwin**
### Related Ticket(s)

#469 Develope Use Cases pattern

use-cases-pattern

### Description
This pull request creates use cases pattern.

Deploy Preview:

### Changelog

**New**

- use cases pattern

### CODE REVIEW

File: packages/patterns-react/src/patterns/UseCases/UseCasesGroup.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        srcSet={`${img.uri.md}, ${img.uri.lg} 2x`}
+      />
+      <source
+        media="(min-width: 320px)"
+        srcSet={`${img.uri.sm}, ${img.uri.md} 2x`}
+      />
+      <img src={img.uri.md} srcSet={`${img.uri.sm}`} alt={img.alt}></img>
+    </picture>
+  );
+};
```
Comment (jeffchew) [inline packages/patterns-react/src/patterns/UseCases/UseCasesGroup.js]: Not urgent for this PR, but I can see this as a standalone component that can be reused.

File: packages/patterns-react/src/internal/FeatureFlags.js [hunk 2]
```diff
@@ -33,9 +33,17 @@ export const LISTSECTION =
...
 export const SIMPLELONGFORM =
   process.env.SIMPLELONGFORM === 'true' || DDS_FLAGS_ALL || false;
+
+/**
+ * This determines if the usecases will be rendered or not
+ *
+ * @type {string | boolean}
+ */
+export const USECASES =
+  process.env.USECASES === 'true' || DDS_FLAGS_ALL || false;
```
Comment (jeffchew) [inline packages/patterns-react/src/internal/FeatureFlags.js]: Per our huddle this morning, let's rename to `DDS_USECASES`

File: packages/patterns-react/src/patterns/UseCases/UseCasesGroup.js (hunk 1 above)
Comment (sannminwin) [reply packages/patterns-react/src/patterns/UseCases/UseCasesGroup.js]: Created new issue.
https://github.com/carbon-design-system/ibm-dotcom-library/issues/699

File: packages/patterns-react/src/internal/FeatureFlags.js (hunk 2 above)
Comment (sannminwin) [reply packages/patterns-react/src/internal/FeatureFlags.js]: updated.

**Comment by ltitle** [comment]
It looks like the title and subheads are not using the right text sizes. The section title should be $expressive-heading-05 and the subheads should be $expressive-heading-04. 

It is hard to tell but also please confirm that the image is spanning 8 columns on desktop.

I do not see the card CTAs for the section. Are you coding that into this pattern?

Vertical spacing looks right on desktop but needs to be reduced for medium and small breakpoints (see specs)

**Comment by jeffchew** [comment]
> It looks like the title and subheads are not using the right text sizes. The section title should be $expressive-heading-05 and the subheads should be $expressive-heading-04.
> 
> It is hard to tell but also please confirm that the image is spanning 8 columns on desktop.
> 
> I do not see the card CTAs for the section. Are you coding that into this pattern?
> 
> Vertical spacing looks right on desktop but needs to be reduced for medium and small breakpoints (see specs)

Thank you @ltitle , @sannminwin can you look into these?

@ltitle I am thinking of merging this as-is and then these feedback can be addressed in a separate PR, just so that the DPP Pilot team can be unblocked for this pattern. Think that would be ok?

**Comment by sannminwin** [comment]
> It looks like the title and subheads are not using the right text sizes. The section title should be $expressive-heading-05 and the subheads should be $expressive-heading-04.
> 
> It is hard to tell but also please confirm that the image is spanning 8 columns on desktop.
> 
> I do not see the card CTAs for the section. Are you coding that into this pattern?
> 
> Vertical spacing looks right on desktop but needs to be reduced for medium and small breakpoints (see specs)

Hi Lila,
Thanks for having a look.
There were 4 versions of specs. This pattern is updated based on the latest version of specs that you provided. Headings are coded as mentioned in specs. Card Link is there  but it seems not rendering because of feature flag bug. I will try to fix now. Here I will include some screenshots of code for typography that you mention. Please let me know if you see any specific discrepency so that we can address them as new issues in the future.

.
xlg-1312px-use case section

Screen Shot 2019-11-19 at 9 56 18 AM

Screen Shot 2019-11-19 at 9 57 03 AM

**Comment by jeffchew** [comment]
@ltitle updates appear to be in now, can you double check? If all looks good we will merge in.

**Pull request merged**